script/update.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -v            # print commands before execution, but don't expand env vars in output
+set -o errexit    # always exit on error
+set -o pipefail   # honor exit codes when piping
+set -o nounset    # fail on unset variables
+
+git clone "https://electron-bot:$GH_TOKEN@github.com/electron/electron-api-historian" module
+cd module
+git submodule update --init
+npm ci
+
+git config user.email electron@github.com
+git config user.name electron-bot
+
+# Update the electron submodule
+pushd electron && git checkout origin/master && popd
+# bail if the submodule sha didn't change
+if [ "$(git status --porcelain -- electron)" = "" ]; then
+  echo "electron origin/master ref has not changed; goodbye!"
+  exit 78
+fi
+git add electron
+ELECTRON_SHA=$(git submodule status --cached electron | awk '{print $1}')
+git commit -m "chore: update to latest electron ($ELECTRON_SHA)"
```

Comment:
? it's not should create one commit or it's something special case here

Reply:
I'm not sure I understand your question, but this will update the repository so that the `index.json` and the sha pointed to by the submodule will match. Mostly just a bookkeeping thing.

Reply:
The question here it's why creating 2 commits instead of 1 where one it's updated electron and where second that contains the `feat` for creating the release, with updated build data.

> Mostly just a bookkeeping thing.

oki-doki

Reply:
Gotcha, yes that makes sense